cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -943,28 +943,86 @@ class CompareNEInstruction extends CompareInstruction {
   }
 }
 
-class CompareLTInstruction extends CompareInstruction {
+class RelationalInstruction extends CompareInstruction {
```

Comment:
I know that many of the classes in this file are missing QLDoc comments, but please add QLDoc for the new class and its abstract methods.

Reply:
Added documentation